e2e-tests/cypress/e2e/pages/insurance/your-buyer.js

```diff
@@ -0,0 +1,20 @@
+import { FIELDS } from '../../../../content-strings/fields/insurance/your-buyer';
+
+const {
+  YOUR_BUYER: {
+    BUYER_COUNTRY: {
+      ID,
```

Comment:
Not a blocker for this PR but in the next PR, will need to use `FIELD_iDS` instead of `FIELDS` because they'll be multiple field ids that need to be consumed